vm/vmimpl/vmimpl.go

```diff
@@ -96,6 +96,11 @@ type BootError struct {
 }
 
 func MakeBootError(err error, output []byte) error {
+	if len(output) == 0 {
+		// In reports, it may be helpful to distinguish the case when the boot output
+		// was collected, but turned out to be empty.
+		output = []byte("<empty boot output>")
```

Comment:
How do we know that the `output` here is true kernel output? Could we lost it before this point?

Reply:
I can move it a bit higher up (before the call to `vmimpl.WaitForSSH`), but it still doesn't guarantee that the code that collects the boot output has no bugs, right?

Reply:
Right. That's why I am asking. Will it still be useful?

Reply:
When I open https://syzkaller.appspot.com/bug?id=a684ec59a0039be484c1d812e68c5e69fd25e7b4, I see no boot output and it's unclear to me whether
1) We did not intend to collect or collected, but did not pass it over to the dashboard.
2) We collected it, but it was empty.

(Assuming there are no bugs in the collecting code), if I saw `<empty boot output>`, it would have been clear to me that we did intend to collect it, but there was no output.